.github/workflows/test-db-versions-minimal-gpu.yml

```diff
@@ -0,0 +1,38 @@
+name: Test Docker GPU
+
+on:
+  workflow_call:
+
+jobs:
+  get_exasol_versions_with_gpu:
+    uses: ./.github/workflows/get-exasol-versions.yml
+    with:
+      only_gpu: true
+  test-docker-gpu:
+    name: Run GPU Minimal Tests for Exasol  ${{ matrix.exasol_version }}
+    runs-on:
+      labels: int-linux-x64-4core-gpu-t4-ubuntu24.04-1
+    needs: get_exasol_versions_with_gpu
+    strategy:
+      fail-fast: false
+      matrix:
+        exasol_version: ${{fromJson(needs.get_exasol_versions_with_gpu.outputs.matrix)}}
+    steps:
+      - name: SCM Checkout
+        uses: actions/checkout@v4
+
+      #Cannot use exasol/python-toolbox/.github/actions/python-environment here
+      #because pipx is not installed on int-linux-x64-4core-gpu-t4-ubuntu24.04-1
```

Comment:
And why do we not install pipx?

Reply:
I tried to install via `apt` but then the PTB threw some strange errors....
see https://github.com/exasol/integration-test-docker-environment/actions/runs/14513334916/job/40716869848

Reply:
I have a guess, you installed pipx into the wrong python environment

Reply:
My guess actually that the steps in https://github.com/exasol/python-toolbox/blob/main/.github/actions/python-environment/action.yml should be inverted -> First install Python, and then run pipx

Reply:
Add a PTB issue to be tackled at another time:
https://github.com/exasol/python-toolbox/issues/408